docs/vcpkg_tool_release_process.md

```diff
@@ -43,3 +43,4 @@ such as https://github.com/microsoft/vcpkg/pull/23757
 13. Merge the tool update PR.
 14. Change the github release in vcpkg-tool from "prerelease" to "release". (This automatically
   updates the aka.ms links)
+15. Follow [these steps](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/vscode-embedded-tools?path=/docs/updating-vcpkg.md&_a=preview) (Microsoft internal) to update the bundled version of vcpkg in the Embedded Tools VS Code extension.

```

Comment:
I think this would be better as the previous step but no big deal (isn't like these are always done in order anyway).